### PR TITLE
support more decimal and thousands separators;

### PR DESCRIPTION
**Pull request by linzijia07**

### CODE REVIEW

File: Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs [hunk 1]
```diff
@@ -74,6 +74,47 @@ public virtual List<ExtractResult> Extract(string source)
 
             return result;
         }
+
+        public Regex AddArabicNumberRegex(ArabicType type, string placeholder = @"\D|\b")
```
Comment (yiylin) [inline Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs]: Renaming this method to GenerateArabicNumberRegex is better. And this method should be protected

File: Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+using System.Globalization;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Immutable;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.Number.Tests
+{
+    public class ArabicNumberTestConfiguration : INumberParserConfiguration
+    {
```
Comment (yiylin) [inline Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs]: It seems that we do not need to create a new configuration.We can use https://github.com/Microsoft/Recognizers-Text/blob/master/Microsoft.Recognizers.Text.Number/English/Parsers/EnglishNumberParserConfiguration.cs

File: Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs (hunk 1 above)
Comment (linzijia07) [inline Microsoft.Recognizers.Text.Number/Extractors/BaseNumberExtractor.cs]: done

File: Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs (hunk 2 above)
Comment (linzijia07) [inline Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs]: We need this new configuration to go through all potential combinations of decimal separator and non-decimal separator. EnglishNumberParserConfiguration just uses one combination.

**Pull request merged**